agent/packaging/ubuntu_xenial/kontena-agent/lib/systemd/system/docker.service.d/50-kontena.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment='DOCKER_OPTS=--insecure-registry="10.81.0.0/19"'
```

Comment:
Seems like the systemd service unit in the `docker-engine=1.11*` package doesn't know about `DOCKER_OPTS`:

`systemctl cat docker.service`
```
# /lib/systemd/system/docker.service
...
[Service]
...
ExecStart=/usr/bin/docker daemon -H fd://
```


`apt policy docker-engine`
```
docker-engine:
  Installed: 1.11.2-0~xenial
  Candidate: 1.12.2-0~xenial
```